website/docs/compute/local-deployment/local-deployment.md

```diff
@@ -0,0 +1,219 @@
+# Local Deployment
+
+It’s possible to deploy a Compute environment to a local environment using a local Kubernetes cluster.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [ngrok](https://ngrok.com/download)
+- [subo](https://github.com/suborbital/subo#installing)
+
+## Steps
+
+### 1. Create a folder for the environment
+
+This is a temporary place where we’ll create and configure our Compute environment:
+
+```bash
+mkdir my-compute; cd my-compute
+```
+
+### 2. Start up our Kubernetes cluster
+
+Kubernetes clusters usually live on the cloud. However, with minikube, we can create a local one to use:
+
+```bash
+minikube start
+```
+
+### 3. Expose our cluster to the internet with ngrok
+
+This command will forward all requests to a randomly-generated URL to `http://localhost:80`
+
+```bash
+ngrok http http://localhost
+```
+
+> **Note:** Jot down that URL generated by ngrok! It’ll look something like [https://84925795ffae.eu.ngrok.io](https://84925795ffae.eu.ngrok.io/)
```

Comment:
This is great, so it works with the free tier! :heart:

Reply:
YESSSS